### PR TITLE
add abstract and implement textual edits in preparation for archive submission

### PR DESCRIPTION
**Pull request by msneubauer**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex [hunk 1]
```diff
@@ -18,7 +18,7 @@
 \usepackage{lineno}  % for line numbering during review
 \linenumbers
 
-\abstract{To be written}
+\abstract{At the heart of experimental high energy physics (HEP) is the development of facilities and instrumentation that provide sensitivity to new phenomena. Our understanding of Nature at is most fundamental level is advanced through the analysis and interpretation of data from sophisticated detectors in HEP experiments.  The goal of data analysis systems is to reduce the “time-to-insight” while exploiting the maximum possible scientific potential of the data within the constraints of computing and human resources. To achieve this goal, future analysis systems should empower physicists to access the data with a high level of interactivity, reproducibility and throughput capability. As part of the HEP Software Foundation’s Community White Paper process, a working group on Data Analysis and Interpretation was formed to assess the challenges and opportunities in HEP data analysis and develop a roadmap for activities in this area over the next decade. In this report, the key findings and recommendations of the Data Analysis and Interpretation Working Group are presented.}
```
Comment (eduardo-rodrigues) [inline CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex]: "at its most"

Comment (eduardo-rodrigues) [reply CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex]: Otherwise abstract looks good!

**Comment by eduardo-rodrigues** [comment]
I spotted a trivial thingy in the abstract. Otherwise it all looks good from a quick browse!

**Comment by msneubauer** [comment]
Thanks, Eduardo. I fixed the abstract typo.

**Pull request closed without merging**